### PR TITLE
Update nidigital, nidmm, nifake, niscope from latest exports

### PR DESCRIPTION
**Pull request by dmondrik**
### What does this Pull Request accomplish?

Updates metadata using actual hapigen exports:
```
nidigital_grpc_device 23.0.0d41
nidmm_grpc_device 23.0.0d46
nifake_grpc_device 23.0.0d49
niscope_grpc_device 23.0.0d58
```

### Why should this Pull Request be merged?

[Task 2186878](https://dev.azure.com/ni/DevCentral/_workitems/edit/2186878): [grpc-device] Uptake new NI-Digital Pattern Driver Export
[Task 2186879](https://dev.azure.com/ni/DevCentral/_workitems/edit/2186879): [grpc-device] Uptake new NI-DMM Export
[Task 2186881](https://dev.azure.com/ni/DevCentral/_workitems/edit/2186881): [grpc-device] Uptake new NI-SCOPE Export

### What testing has been done?

Built locally.

### CODE REVIEW

**Comment by reckenro** [comment]
Ah forgot ni-fake will need some updates. I think it's just in `source/tests/unit/ni_fake_service_tests.cpp` where all instances of `set_requested_behavior` needs to be updated to `set_initialization_behavior`.

File: generated/niscope/niscope.proto [hunk 1]
```diff
@@ -465,8 +465,8 @@ enum ExportableSignals {
   EXPORTABLE_SIGNALS_NISCOPE_VAL_READY_FOR_ADVANCE_EVENT = 6;
   EXPORTABLE_SIGNALS_NISCOPE_VAL_READY_FOR_START_EVENT = 7;
   EXPORTABLE_SIGNALS_NISCOPE_VAL_READY_FOR_REF_EVENT = 10;
-  EXPORTABLE_SIGNALS_NISCOPE_VAL_REF_CLOCK = 100;
   EXPORTABLE_SIGNALS_NISCOPE_VAL_5V_OUT = 13;
+  EXPORTABLE_SIGNALS_NISCOPE_VAL_REF_CLOCK = 100;
```
Comment (reckenro) [inline generated/niscope/niscope.proto:469]: Note to self: I keep seeing this ref_clock one jump around. Guessing it's something goofy in hapigen metadata causing this with every update. Will look into.

**Review by reckenro — APPROVED** [review]
Just need a small fixup in the ni_fake_service_tests for the name change.

**Review by astarche — APPROVED** [review]

File: generated/niscope/niscope.proto (hunk 1 above)
Comment (dmondrik) [reply generated/niscope/niscope.proto:469]: I'm guessing it's not explicitly sorted when it should be? This time it appears they happen to be sorted numerically. I'm going to mark it resolved and see if we can fix that up separately.

Comment (reckenro) [reply generated/niscope/niscope.proto:469]: Yeah.. it's re-arranged the same way in enums.py so I'm guessing it's something in the hapigen metadata (probably something to do with 'position' key on that end)

**Pull request merged**